docker/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM foundationdb/foundationdb-build:0.0.2
-LABEL version=0.0.3
-LABEL description="This image contains all the dependecies for FoundationDB Document Layer CI. \
+FROM foundationdb/foundationdb-build:0.0.5
+LABEL version=0.0.4
+LABEL description="This image contains all the dependencies for FoundationDB Document Layer CI. \
 For more details on how to use this image look at project README file at \
 https://github.com/FoundationDB/fdb-document-layer"
 
 RUN mkdir -p /opt && \
     cd /opt && \
-    curl -L -J https://cmake.org/files/v3.12/cmake-3.12.1-Linux-x86_64.tar.gz | tar -xz && \
+		rm -rf /opt/boost_1_67_0 && \
```

Comment:
Why are we not reusing the boost coming from FoundationDB docker image? We could have built the existing boost directory. Is there any issue with that?